dragscroll.js

```diff
@@ -90,3 +95,16 @@
     exports.reset = reset;
 }));
 
+function zoomIn(n) {
```

Comment:
This is a global method, which is bad (someone can redefine it).

This should be an exported method, just like I did with `reset()`.